cms/templates/js/content-group-details.underscore

```diff
@@ -32,7 +32,7 @@
             </li>
         <% } else { %>
             <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Cannot delete when in use by a unit') %>">
-                <button class="delete action-icon is-disabled" aria-disabled="true" disabled="disabled" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
+                <button class="delete action-icon is-disabled" disabled="true" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
```

Comment:
@ssemenova I am still confused by the lack of aria-hidden="True" here, while it is present for the parallel location below. In both cases, these are disabled buttons; the only difference is that one is for content groups, while the other is for group configurations.

You can see both of these at the same time at https://studio-ssemenovadisabledstates.sandbox.edx.org/group_configurations/course-v1:edX+DemoX+Demo_Course

I don't see any reason why they should be different. @clrux, do you have an opinion about whether or not aria-hidden should be present?